payment-circle/src/main/java/org/stellar/anchor/paymentservice/circle/CirclePaymentService.java

```diff
@@ -266,6 +280,95 @@ private HttpClient getWebClient(boolean authenticated) {
             });
   }
 
+  public Mono<CircleTransferListResponse> getTransfers(
+      String accountID, String beforeCursor, String afterCursor, Integer pageSize)
+      throws HttpException {
+    // build query parameters for GET requests
+    int _pageSize = pageSize != null ? pageSize : 50;
+    LinkedHashMap<String, String> queryParams = new LinkedHashMap<>();
+    queryParams.put("pageSize", Integer.toString(_pageSize));
+    queryParams.put("walletId", accountID);
+
+    if (afterCursor != null && !afterCursor.isEmpty()) {
+      queryParams.put("pageAfter", afterCursor);
+      // we can't use both pageBefore and pageAfter at the same time, that's why I'm using 'else if'
+    } else if (beforeCursor != null && !beforeCursor.isEmpty()) {
+      queryParams.put("pageBefore", beforeCursor);
+    }
```

Comment:
In my StellarPaymentService PR I got rid of the `beforeCursor` and only support a single `cursor` parameter. Do you think that could work for the CirclePaymentService?

Reply:
Well Horizon's endpoints only support a single `cursor`, so the issue is that I would have to ignore the `beforeCursor` specified for calls on `StellarPaymentService` objects.

I don't see the need for a `beforeCursor` because that limit can always be imposed by the client consuming the data. This differs from the `afterCursor` which is required to fetch the next page of data.

Reply:
In Horizon, you actually have the `next` and `prev` pages, which work similarly to the `after` and `before` cursors. Although the prev page is returned in inverse order, which can make things a bit messy to translate from one convention to the other.

I'm ok disabling the before cursor though, in order to make implementation simpler. I'd rather do that in a separate PR, if that's ok.